indigo_app/templates/library.html

```diff
@@ -127,6 +127,26 @@ <h3 class="panel-title">Tags</h3>
     {{/each}}
   </div>
 </div>
+
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Status</h3>
+  </div>
+
+  <div class="panel-body">
+    <div class="btn-group" data-toggle="buttons">
+      <label class="btn btn-default active">
+        <input type="radio" name="options" autocomplete="off" class="filter-status" data-status="all" checked>All
```

Comment:
You can store the "all" in the `value` attribute, which is the "right" way to do it for HTML forms. Then you don't need to use `data("status")` in the code, and can just use `val()` instead.